MauldinEconomics/mautic
# UI Form Details: show count of form submission in Form detail page

| Q  | A
| --- | ---
| Bug fix? | N
| New feature? | Y
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | N
| Deprecations? | N

#### Description:

Add form submission count to form details expansion tab.

#### Steps to test this PR:

1. Create a form.
2. Send a few submissions to the form, using a combination of unique and non-unique lead values. This should probably be done _incognito_.
3. View the overview page for that form in the backend, like `<your-domain>/s/forms/view/<form-id>`. Expand the "DETAILS" tab and check the table row labeled **Submissions (unique/total)**.
